network-area-diagram/src/main/java/com/powsybl/nad/svg/DefaultEdgeRendering.java

```diff
@@ -185,20 +185,7 @@ private List<Double> computeLoopAngles(Graph graph, List<BranchEdge> loopEdges,
             } else if (nbLoops <= nbSharedSlots) {
                 // Place the maximum of loops in "slots" separated by non-loop edges, and put the excessive ones in the bigger "slots"
                 int nbExcessiveRemaining = nbLoops - nbSeparatedSlots;
-                for (int i = sortedIndices.size() - 1; i >= 0; i--) {
-                    int iSorted = sortedIndices.get(i);
-                    int nbAvailableSlots = (int) Math.floor(deltaAngles[iSorted] / apertureWithMargin);
-                    if (nbAvailableSlots == 0) {
-                        break;
-                    }
-                    int nbLoopsInDelta = Math.min(nbAvailableSlots, nbExcessiveRemaining + 1);
-                    double extraSpace = deltaAngles[iSorted] - svgParameters.getLoopEdgesAperture() * nbLoopsInDelta; // extra space without margins
-                    double intraSpace = extraSpace / (nbLoopsInDelta + 1); // space between two loops and between non-loop edges and first/last loop
-                    double angleStep = (anglesOtherEdges.get(iSorted + 1) - anglesOtherEdges.get(iSorted) - intraSpace) / nbLoopsInDelta;
-                    double startAngle = anglesOtherEdges.get(iSorted) + intraSpace / 2 + angleStep / 2;
-                    IntStream.range(0, nbLoopsInDelta).mapToDouble(iLoop -> startAngle + iLoop * angleStep).forEach(loopAngles::add);
-                    nbExcessiveRemaining -= nbLoopsInDelta - 1;
-                }
+                computeLoopAnglesWithExcessiveOne(nbExcessiveRemaining, sortedIndices, deltaAngles, apertureWithMargin, svgParameters, anglesOtherEdges, loopAngles);
```

Comment:
As a beginner in powsybl-diagram, this function name was not easy for me to understand at first. 

In order to make this part of the code clearer, I would suggest to:
- encapsulate the part of the code in the `if(nbLoops <= nbSeparatedSlots)` block
```
                for (int i = sortedIndices.size() - nbLoops; i < sortedIndices.size(); i++) {
                    int iSorted = sortedIndices.get(i);
                    loopAngles.add((anglesOtherEdges.get(iSorted) + anglesOtherEdges.get(iSorted + 1)) / 2);
                } 
```
in a function called for example `computeLoopAnglesWhenEnoughSeparatedSlotsPresent(...)`

- And rename the `computeLoopAnglesWithExcessiveOne(...)` with `computeLoopAnglesWhenEnoughSharedSlotsPresent(...)`

What do you think ?